README.md

```diff
@@ -213,6 +213,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Amrithshagar</b></sub>
                 </a>
             </td>   
+            <td align="center">
+                <a href="https://github.com/Amrithshagar">
+                    <img src="https://avatars.githubusercontent.com/u/110282357?v=4" width="100px;" alt="Shino Shaji"/>
+                    <br />
+                    <sub><b>Rinku Singh</b></sub>
+                </a>
```

Comment:
Please keep your card realistic. The alt attribute and your name should be the same, your github handle is: https://github.com/JaydeepRokade